entrypoint.sh

```diff
@@ -76,7 +76,7 @@ if [[ ! -f ${ROOTFS}/etc/systemd/system/kubelet.service ]]; then
 	Documentation=http://kubernetes.io/docs/
 
 	[Service]
-	Environment="KUBELET_VERSION=${K8S_VERSION}_coreos.0"
```

Comment:
Warning ! This change is related to a specific version of the `Kubelet-wrapper`...

The one distributed in CoreOS Stable `1185.3.0` does not work ...